hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/logging/TransactionStateLogger.java

```diff
@@ -235,7 +232,7 @@ public static void logEndTransactionRecord(
      * @param <T> The type of the singleton
      */
     public static <T> void logSingletonRead(@NonNull final String label, @Nullable final ValueLeaf<T> value) {
-        if (logger.isDebugEnabled() && Thread.currentThread().getName().startsWith(HANDLE_THREAD_NAME)) {
+        if (logger.isDebugEnabled() && Thread.currentThread().getName().equals(TRANSACTION_HANDLING_THREAD_NAME)) {
             logger.debug("      READ singleton {} value {}", label, value == null ? "null" : value.getValue());
         }
```

Comment:
This statement will always evaluate to `false` after this PR merges. The thread where we handle transactions is from the wiring framework, meaning that the thread name will never contain the string we are checking against.

IMO, adding strong dependencies on the naming schema of platform threads is a code smell. Platform thread names are not public API, and so it's dangerous to depend on the naming schema. But since this code is preexisting, we should probably double check with the services team of its importance before making it defunct.

Reply:
> This statement will always evaluate to false after this PR merges

This PR changes the value of the constant to match the new name from the framework, so the functionality of this condition doesn't change post merge. 

> adding strong dependencies on the naming schema of platform threads is a code smell

I agree, but I think it's too late for now. This was recently implemented, https://github.com/hashgraph/hedera-services/pull/8633. If we remove the link, it would need to be paired with a replacement solution

Reply:
Ah, ok. This is a sequential thread scheduler, so we actually do set a name for it. I'm with this for the time being.